### DISCUSSION TITLE
Cannot connect to my Minecraft Pterodactyl server

### DISCUSSION THREAD

**Discussion by DenoxTV** [pterodactyl/panel#3827] [Support]
### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.

### Current Behavior

I created a Minecraft Bungeecord network with a spigot server and a Bungeecord server and I have a network error, when I want to connect to my server, the game says me : Connexion Refused. I asked everywhere and nobody has the solution. 

### Expected Behavior

I would be avalaible to connect to my server.

### Steps to Reproduce

I created a servers with the Pterodactyl Panel and I allowed each of them a port. I configured the servers like this article (https://pterodactyl.io/community/games/minecraft.html#configuring-a-server-network-bungeecord-waterfall-hexacord-etc) and i configured them with the port allowed

### Panel Version

1.6.6

### Wings Version

1.5.3

### Error Logs

```bash
https://ptero.co/jiqiqepydo
```

**Comment by trenutoo** [comment]
You'd want to display your configuration and allocations for the servers themselves. The information you've provided so far lacks information for anyone to assist.

Connection refused error itself suggests the port is closed.
